frontend/appflowy_flutter/lib/window/app_window_size_manager.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:convert';
+
+import 'package:shared_preferences/shared_preferences.dart';
+
+class WindowSizeManager {
+  static const double minWindowHeight = 400.0;
+  static const double minWindowWidth = 600.0;
+
+  Future<void> saveSize(
+    double currentWindowHeight,
+    double currentWindowWidth,
+  ) async {
+    final windowSize = {
+      'height': currentWindowHeight < minWindowHeight
+          ? minWindowHeight
+          : currentWindowHeight,
+      'width': currentWindowWidth < minWindowWidth
+          ? minWindowWidth
+          : currentWindowWidth,
+    };
+
+    final prefs = await SharedPreferences.getInstance();
+    await prefs.setString('windowSize', json.encode(windowSize));
+  }
+
+  Future<Map<String, dynamic>> getSize() async {
+    final prefs = await SharedPreferences.getInstance();
+    prefs.remove('windowSize');
```

Comment:
and why to remove the key every time?

Reply:
Removing the key every time was for testing purposes.